lib/6to5/generation/whitespace.js

```diff
@@ -61,14 +79,5 @@ Whitespace.prototype.getNewlinesBetween = function (startToken, endToken) {
   var start = startToken ? startToken.loc.end.line : 1;
   var end   = endToken.loc.start.line;
 
-  var lines = 0;
-
-  for (var line = start; line < end; line++) {
-    if (!_.contains(this.used, line)) {
-      this.used.push(line);
-      lines++;
-    }
-  }
-
-  return lines;
-};
+  return util.mergeIntegerRange(this.used, start, end);
```

Comment:
It's probably best to just make `this.used` an object that we can just lookup as this adds a lot more complexity.